GameOfLife/GameOfLife/Program.cs

```diff
@@ -1,2 +1,17 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using GameOfLife;
+
+Console.WriteLine("Hello! This is Conway's Game of Life.");
+
+Console.WriteLine();
+
+Console.WriteLine("Enter height, please: ");
+string height = Console.ReadLine();
+
+Console.WriteLine("Enter width, please:");
+string width = Console.ReadLine();
+
+Console.Clear();
+
+var board = new Board(int.Parse(height), int.Parse(width));
```

Comment:
Please validate user input, for example if user enter @ or -10

Reply:
Input value validation enabled. Validation class created with appropriate method.